packages/jaeger-ui/src/components/Monitor/ServicesView/index.test.js

```diff
@@ -394,6 +394,6 @@ describe('yAxisTickFormat', () => {
   it('value is 2 second', () => {
     const timeInMs = 1000;
     const displayTimeUnit = 'seconds';
-    expect(yAxisTickFormat(timeInMs, displayTimeUnit)).toBe(1);
+    expect(yAxisTickFormat(timeInMs, displayTimeUnit)).toBe('1');
```

Comment:
The snapshot above, packages/jaeger-ui/src/components/Monitor/ServicesView/__snapshots__/serviceGraph.test.js.snap, is very large and not useful. Can we rethink the unit tests that produces it and replace it with more explicit tests for user-facing features instead of a snapshot?

Reply:
Yes, instead of relying on snapshots, we can write individual test cases that cover specific scenarios and assertions. Thank you.

Reply:
I've replaced the snapshot test with more explicit test and also added test for tooltip formatter. Can you check it once?